pkg/client/connector/service.go

```diff
@@ -504,9 +505,16 @@ func run(c context.Context) error {
 		if !ok {
 			return nil
 		}
-		tm.sshPortForward(c,
-			"-D", "localhost:1080",
-		)
+		ln, err := net.Listen("tcp4", "127.0.0.1:0")
```

Comment:
Is there a specific reason why you used `tcp4`/`127.0.0.1` instead of maybe-IPv6 `tcp`/`localhost`?

Reply:
I think using `localhost` consumes more CPU-cycles since that name must be resolved somehow. Do you see that doing so would bring any advantages?